administrator/components/com_admin/sql/updates/mysql/5.1.0-2024-02-10.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `#__guidedtour_steps` ADD COLUMN `params` text NOT NULL /** CAN FAIL **/;
```

Comment:
```suggestion
ALTER TABLE `#__guidedtour_steps` ADD COLUMN `params` text NULL /** CAN FAIL **/;
```
We shouldn't use the NOT NULL constraint for the new columns as that can make problems for updates, or at least can make it more complicated on PostgreSQL. The `#__categories` table also allows null values for its `params` column, so we would be consistent with that.

Reply:
Thanks Richard